sway-fmt-v2/CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contributing to the Sway Language Formatter
+
+Firstly, thank you for taking interest in advancing the Sway language formatter! This guide will walk you through installation and best practices for contributing to this project.
+
+## Installation
+
+In order to see `swayfmt` in action you will need both `swayfmt` and the [`forc-fmt`](../forc-plugins/forc-fmt/) plugin. As previously stated, you can install these via [`fuelup`](https://github.com/FuelLabs/fuelup) which will install the complete Fuel toolchain, or follow the below instructions to install from source.
+
+**note:** `cargo` is a pre-requisite to this build
+
+1. move to your preferred directory
+2. clone the Sway repo
+3. build from manifest and move the compiled result to your `.cargo/bin` folder
+
+```sh
+git clone https://github.com/FuelLabs/sway.git
+cargo build --manifest-path ~/sway/forc-plugins/forc-fmt/Cargo.toml
+mv ~/sway/target/debug/forc-fmt ~/.cargo/bin
+```
+
+### Testing
+
+Simply run the `forc-fmt` command on the file you want to format:
```

Comment:
`sway-fmt-v2` is currently used by `forc-fmt-v2` and we are still distributing `forc-fmt` (which uses `sway-fmt`) with `fuelup`. So our users will have `forc-fmt` installed which can create a confusion here. Considering this is the readme for `sway-fmt-v2` we may want to consider changing this to `forc-fmt-v2` until we make the complete transition and replace/remove v1 from distribution.

Reply:
Since it's getting close to being switched over I've left it like this, but I can add a line that is explicit about that too